web/src/engine/websites/DigitalTeam.ts

```diff
@@ -0,0 +1,52 @@
+import { Tags } from '../Tags';
+import icon from './DigitalTeam.webp';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { Fetch, FetchRequest } from '../FetchProvider';
+
+@Common.MangaCSS(/^https?:\/\/dgtread\.com\/[^/]+/, 'div#manga_right div.title')
+@Common.MangasSinglePageCSS('/reader/series', 'div#series_list ul li.manga_block ul li.manga_info div.manga_title a')
+@Common.ChaptersSinglePageCSS('div.chapter_list ul li div.ch_top a')
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('digitalteam', `DigitalTeam`, 'https://dgtread.com', Tags.Language.Italian, Tags.Media.Manga, Tags.Source.Scanlator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        //First : fetch page
+        let uri = new URL(chapter.Identifier, this.URI);
+        let request = new FetchRequest(uri.href);
+        let response = await Fetch(request);
+        const dat = await response.text();
+        const external = dat.includes('js/jq_rext.js');
+
+        uri = new URL('/reader/c_i', this.URI);
+        request = new FetchRequest(uri.href, {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/x-www-form-urlencoded'},
+            body: new URLSearchParams({
+                'info[manga]': chapter.Parent.Identifier.split('/').pop(),
+                'info[chapter]': chapter.Identifier.split('/').slice(-2)[0],
+                'info[ch_sub]': '0',
+                'info[title]': 'Digital Team'
+            })
+        });
+        response = await Fetch(request);
+        let data = await response.json();

```

Comment:
Make a type definition of the expected JSON structure to avoid `any` types.
See [MangaDex](https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/MangaDex.ts#L141) example.

Reply:
I thought about it but how would i create a type with unnamed arrays? 
![image](https://user-images.githubusercontent.com/10671518/214250505-eea98e50-b72c-4abc-a2a3-4eee9fffa396.png)

Reply:
Use `unknown` itself or Unions, or maybe both...
```typescript
type APIResult = {
    id: string;
    title: string;
} | string | unknown;
type APIResuts = APIResult[];
```

Reply:
You don't have to mirror everything, you only need to specify the type with the properties you are going to use

Reply:
If you know the exact types for the array index you may try array spread
`type APIResult = [ { id: string, title: string }, string[], string ];`